src/internal/views/nunjucks/billing/batch-list.njk

```diff
@@ -29,20 +29,20 @@
         {% for batch in batches %}
         <tr class="govuk-table__row">
           <td class="govuk-table__cell">
-            <a class="govuk-link" href="/waiting/{{batch.eventId}}">
+            <a class="govuk-link" href="/billing/batch/{{batch.id}}/summary">
```

Comment:
Hi Mark - if we change this link to point directly to the summary page and the batch is still processing it will bypass the waiting screen and will fail to get the details of the batch then also the UI will not be displayed correctly. Unless if we add an If batch status is not processing then use /billing/batch.... else /waiting/...